docs/docs/remote.md

```diff
@@ -91,12 +91,15 @@ Replace the followings variables:
         -p $XPRA_EXTERNAL_PORT:10000 \
         -e CONNECTION_METHOD=xpra \
         -e RMT_PASSWORD=$RMT_PASSWORD \
+        -e XPRA_SUBPATH="/" \
         $BASE_IMAGE/remote
     ```
 
+    Set the `XPRA_SUBPATH` to the subpath that `xpra` should serve on. If you want to have it running on `/xpra`, set `XPRA_SUBPATH` to `/xpra`.
+
     Then, open a browser and connect to:
     ```
-    http://techuser:${RMT_PASSWORD}@localhost:${XPRA_EXTERNAL_PORT}?floating_menu=0
+    http://techuser:${RMT_PASSWORD}@localhost:${XPRA_EXTERNAL_PORT}${XPRA_SUBPATH}/?floating_menu=0
```

Comment:
Is the `/` after `${XPRA_SUBPATH}` correct here? Doesn't the `rewrite` rule change this to `//?floating_menu=0` in case that there is a `/` at the end of the subpath?

Reply:
I suppose that I have to change `/$1` to `$1` anyway. Have to try it tomorrow.

Reply:
While trying out xpra for another PR, I had a look into the documentation and found some inconsistency: I would suggest either rename `${XPRA_EXTERNAL_PORT}` here to simply `${XPRA_PORT}`, as used in the Makefile or rename it in the makefile. However, since it should be clear that the port in the Makefile is the external port, I would probably just remove the `EXTERNAL` here.

Reply:
I tried to change the `/$1` to `$1`, but this doesn't work unfortunately, it leads to an invalid nginx configuration. 

Also, the `/` after XPRA path is needed:
- When XPRA_SUBPATH is empty (``) or `/`, the `/` in the URL is not needed, but doesn't hurt.
- When XPRA_SUBPATH is `/xpra`, the `/` is needed, (`/xpra/` connects properly, but `/xpra` doesn't).